onto_tool/onto_tool.py

```diff
@@ -404,6 +405,45 @@ def generateGraphic(fileRefs, compact, output, version):
     og.create_graf()
 
 
+def __perform_export__(output, output_format, paths, context=None,
+                       strip_versions=False,
+                       merge=None,
+                       defined_by=None, retain_defined_by=False):
+    """Export one or more files as a single output.
+
+    Optionally, strips dependency versions and merges ontologies into
+    a single new ontology.
+    """
```

Comment:
I'm assuming your following the [PEP 257 docstring convention](https://www.python.org/dev/peps/pep-0257/) across this file.

I would just add a description and type for each parameter on this function. Possibly some other functions in this file would be better off with documentation of their parameters, but I'll let you make that call.

One other thing, I often find myself doing this way more since I learned about it. You can add [type hints](https://docs.python.org/3/library/typing.html) to variables/parameters in your code. It doesn't get enforced, just makes it easier find where a variable gets declared and to know what value it's expected to be, totally not required.
```python
# Example type hints
def __perform_export__(output:str, output_format:str, paths:list, context:str=None, ...
```

Reply:
I added documentation to the method. I do try to use docstrings throughout, though most are fairly terse, and I maintain more extensive documentation in the `README.md`. 

I have not done type declarations, as python is technically not a strongly-typed language, and having type declarations would imply more than then language guarantees.